Components/Services/PaymentService.php

```diff
@@ -784,6 +787,10 @@ public function sendOrder($mollieId)
         try {
             /** @var \Mollie\Api\Resources\Order $mollieOrder */
             $mollieOrder = $this->apiClient->orders->get($mollieId);
+
+            if ($mollieOrder->isShipping()) {
+                return (new ArrayCollection($mollieOrder->shipments()->getArrayCopy()))->last();
```

Comment:
hi, just to verify this...the code below with saveTransactionAsShipped looks good, but does this?

Reply:
this is basicly only returning the last shipment, that gets found, because you can not ship things, that are already shipped, I'd say

Reply:
ah ok and shippings get returned..ok :)